Dockerfile

```diff
@@ -15,17 +13,43 @@ RUN mix local.rebar --force
 RUN mix deps.get --only-prod
 RUN mix deps.compile
 
-# Build
+
+### Node Deps and Build Frontend
+FROM node:20.13.1-alpine3.19 AS node
+
+# npm deps
+ENV NODE_ENV=production
+COPY ./package.json /app/package.json
+COPY ./package-lock.json /app/package-lock.json
+WORKDIR /app
+RUN npm ci
+
+# build frontend
+COPY --from=elixir-deps /app/deps /app/deps
+COPY ./tailwind.config.ts /app/tailwind.config.ts
+COPY ./js /app/js
+COPY ./css /app/css
+COPY ./lib/orbit_web /app/lib/orbit_web
+WORKDIR /app
```

Comment:
**question:** is this redundant with the `WORKDIR` on line 24? Or is there intentional nesting of `/app` directories or something?

Reply:
I just noticed that this `COPY WORKDIR RUN COPY WORKDIR RUN` pattern is the same as we do in Glides. Which doesn't mean that it's right, just that it's not completely broken.
https://github.com/mbta/glides/blob/4e7fb2e70d52251fd93b25df1e3a0d9428fd3243/Dockerfile#L45

Reply:
Discussion: https://mbta.slack.com/archives/GP5QJM98A/p1716495046574599
The COPY isn't broken because the first arg is on the build host, and the 2nd arg is in the container.
The extra WORKDIR is redundant, like doing `cd` twice. But I don't think it's a problem, it's kind of nice to have the working directory listed before each RUN, it makes each `COPY WORKDIR RUN` block a little more standalone.